src/Tribe/Updater.php

```diff
@@ -39,7 +39,7 @@ public function do_updates() {
 				}
 			}
 
-			foreach ( $this->constant_updates() as $callback )  {
+			foreach ( $this->get_constant_update_callbacks() as $callback )  {
```

Comment:
* :no_entry_sign: Expected exactly one space between closing parenthesis and opening control structure; "  " found.
* :no_entry_sign: Expected 1 space(s) after closing parenthesis; found 2

<sub>hash: 120329653b12a9b65d124fb15c4b0cf9</sub>